docs/Contributing.md

```diff
@@ -128,6 +128,26 @@ useful as a comment when reading the code? Or does it only distract?
 Is the explanation independent of the current context and is it likely
 to stand the test of time?
 
+## Write useful tests
+
+New code requires new tests to be written that will exercise that code.  This
+applies to bug fixes as well as new features.  The tests serve two purposes:
+ensure that the new code does not regress if it needs to be changed and to give 
+an example of what the new code does.
+
+The tests must exist in the LLPC repository.  They can be a 
+[shaderdb test](../llpc/test/shaderdb), an [LGC test](../lgc/test), or a 
+[unit test](../lgc/unittests).  We do not want to rely on the Vulkan CTS because
```

Comment:
Can you also add a link to llpc unit tests?
I'd also consider turning this inline list of tests into bullet points.